src/Microsoft.AspNet.Testing.Logging/project.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+    "version": "1.0.0-*",
+    "dependencies": {
+        "Microsoft.Framework.Logging": "1.0.0-*"
+    },
+    "frameworks" : {
```

Comment:
Also need `net45` in here? The rest of Logging supports `net45` as well, e.g. https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Framework.Logging/project.json#L13

@bricelam in case he has thoughts on this.